### PR TITLE
 hook up KBFS upgrade to migration CORE-8941

### PR DESCRIPTION
**Pull request by mmaxim**
Hooks up KBFS to the automatic TLF upgrade process (disabled for KBFS on the server), and add a CLI command to manually upgrade TLFs.

cc @strib

### CODE REVIEW

**Comment by mmaxim** [comment]
Probably.

**Comment by maxtaco** [comment]
Ok, cc @strib, halp!

**Comment by mmaxim** [comment]
Actually this is probably good to merge before all the problems are worked out, at least other people could try it (no one will be hurt by this). @maxtaco PTAL!

File: go/teams/teams.go [hunk 1]
```diff
@@ -384,6 +384,34 @@ func (t *Team) AllApplicationKeys(ctx context.Context, application keybase1.Team
 	return AllApplicationKeys(t.MetaContext(ctx), t.Data, application, t.chain().GetLatestGeneration())
 }
 
+func (t *Team) AllApplicationKeysWithKBFS(ctx context.Context, application keybase1.TeamApplication) (res []keybase1.TeamApplicationKey, err error) {
+	teamKeys, err := t.AllApplicationKeys(ctx, application)
+	if err != nil {
+		return res, err
+	}
+	kbfsKeys := t.KBFSCryptKeys(ctx, application)
```
Comment (mmaxim) [inline go/teams/teams.go]: @maxtaco Are you ok with this update to this PR? This implements what we talked about last night.

File: go/teams/teams.go [hunk 2]
```diff
@@ -384,6 +384,34 @@ func (t *Team) AllApplicationKeys(ctx context.Context, application keybase1.Team
...
+			res = append(res, keybase1.TeamApplicationKey{
+				Application:   application,
+				KeyGeneration: keybase1.PerTeamKeyGeneration(k.KeyGeneration),
+				Key:           k.Key,
+			})
+		}
+		for _, tk := range teamKeys {
+			res = append(res, keybase1.TeamApplicationKey{
+				Application:   application,
+				KeyGeneration: keybase1.PerTeamKeyGeneration(tk.Generation() + latestKBFSGen),
```
Comment (maxtaco) [inline go/teams/teams.go]: Are you sure this isn't off-by-one?  My formula last night had a `-1` or `+1`, I forget...

**Pull request merged**